src/api/bot.ts

```diff
@@ -24,6 +24,9 @@ export interface Config {
   token: string;
   dbUrl: string;
   commandPath: string;
+  slashCommandPath: string;
```

Comment:
all of these paths are a bit cluttering now.. should prob move somewhere else at a later time